cmd/sealer/cmd/cluster/apply.go

```diff
@@ -175,7 +175,9 @@ func createNewCluster(clusterImageName string, infraDriver infradriver.InfraDriv
 	if applyMode == common.ApplyModeLoadImage {
 		logrus.Infof("start to apply with mode(%s)", applyMode)
 
-		if err = distributor.DistributeRegistry(infraDriver.GetHostIPList()[0], infraDriver.GetClusterRootfsPath()); err != nil {
+		cluster := cf.GetCluster()
```

Comment:
what's the difference between `cf.GetCluster().GetMaster0IP()` and `infraDriver.GetHostIPListByRole(common.MASTER)[0]`